expression/builtin_compare_vec_test.go

```diff
@@ -34,6 +34,7 @@ var vecBuiltinCompareCases = map[string][]vecExprBenchCase{
 			gener{defaultGener{eType: types.ETDecimal, nullRation: 0.2}},
 			gener{defaultGener{eType: types.ETDecimal, nullRation: 0.2}},
 		}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt, types.ETInt}},
```

Comment:
Please consider setting [`vecExprBenchCase.childrenFieldTypes`](https://github.com/pingcap/tidb/blob/a22af3f821b2798285f2ae98b23f22de7dfde8b5/expression/bench_test.go#L453) to cover the unsigned cases.

Reply:
done.